src/ansi-c/c_main.cpp

```diff
@@ -81,8 +81,7 @@ bool c_main(
 
   if(matches.empty())
   {
-    messaget message(message_handler);
-    message.error("main symbol `" + themain + "' not found");
+    std::cout << "main symbol `" << themain << "' not found" << std::endl;
```

Comment:
shall we use the fmt library?

Reply:
Fixed and replaced them with the standard message handler. However, using the string concatenation instead of fmt. 

Tried to use fmt headers in the old frontend, but compiler failed to locate them. I've ask @rafaelsamenezes for the most elegant way to include fmt in old frontend.

Reply:
The fmt library does not link in old frontends. 
Since the string concatenation is a standard C++ operation, I guess it's ok to use the string concatenation instead of fmt.